Dockerfile

```diff
@@ -13,8 +13,10 @@ RUN echo 'deb https://deb.nodesource.com/node_8.x xenial main' > /etc/apt/source
 
 COPY ./setup/docker/apt_requirements.txt ${ENKETO_SRC_DIR}/setup/docker/
 WORKDIR ${ENKETO_SRC_DIR}/
+# From `man dpkg`, "confdef: If a conffile has been modified and the version in
+# the package did change, always choose the default action without prompting."
 RUN apt-get update && \
-    apt-get upgrade -y && \
+    apt-get upgrade -y -o Dpkg::Options::="--force-confdef" && \
```

Comment:
`man dpkg` also says `If there is no default action it will stop to ask the user unless --force-confnew or --force-confold is also been given`. 

I guess the options should be `apt-get upgrade -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold"`.

Reply:
According to the title of the issue, I would have thought the answer has been "yes".